src/toasts/AnalyticsToast.tsx

```diff
@@ -136,7 +136,7 @@ export const showPseudonymousAnalyticsOptInToast = (analyticsOptIn: boolean): vo
         // The user had no analytics setting previously set, so we just need to prompt to opt-in, rather than
         // explaining any change.
         const learnMoreLink = (sub) => (
-            <AccessibleButton kind="link" onClick={onLearnMoreNoOptIn}>{ sub }</AccessibleButton>
+            <AccessibleButton kind="link_inline" onClick={onLearnMoreNoOptIn}>{ sub }</AccessibleButton>
```

Comment:
See below: `"Share anonymous data to help us identify issues. Nothing personal. No third parties. " +
                "<LearnMoreLink>Learn More</LearnMoreLink>", {}, { "LearnMoreLink": learnMoreLink }),`